elyra/pipeline/kfp/kfp_processor.py

```diff
@@ -451,8 +437,8 @@ def process(self, pipeline):
             object_storage_path = None
 
         return KfpPipelineProcessorResponse(
-            run_id=run.id,
-            run_url=f"{public_api_endpoint}/#/runs/details/{run.id}",
+            run_id=run.run_id,
+            run_url=f"{public_api_endpoint}/{experiment.experiment_id}/runs/{run.run_id}",
```

Comment:
This explicit change for ODH dashboard, not sure if it would work with kubeflow pipeline UI.
https://www.kubeflow.org/docs/components/pipelines/legacy-v1/overview/quickstart/#deploy-kubeflow-and-open-the-kubeflow-pipelines-ui